### DISCUSSION TITLE
Clearing forms using the reset method

### DISCUSSION THREAD

**Discussion by Coachbre** [nss-day-cohort-47/C47-Discussions#50] [Look what I learned!]
Autumn and Laurel helped me figure out how to clear the giffygram entry form after clicking save! 

I created a clearForm function that uses the reset() method, with the ID matching the ID assigned to the form in the PostEntry module, then imported it to main.js and invoked clearForm within both the 'cancel and 'save' eventListeners

*once you change the form attribute from a class to an ID, you may want to do the same in post.css

![postEntry](https://user-images.githubusercontent.com/62270575/110846302-a2dd5500-8279-11eb-91d0-02bee938877c.PNG)
![main](https://user-images.githubusercontent.com/62270575/110846337-aa9cf980-8279-11eb-929d-38482810eb4e.PNG)

**Comment by Chadwell-Clark** [comment]
I did something similar to reset the placeholder text both when I cancel a post entry and also after a post entry is made. I called showPostEntry() in both instances.

**Reply by Chadwell-Clark** [reply]
I will try this on my journal this weekend. Thank you!

**Comment by AutumnFoto** [comment]
so glad we figured it out! 🥇 👍
